UNR-2317 Fix crash when starting PIE with missing schema (#1481)
spatialos/UnrealGDK: CHANGELOG.md

- Replicating a static subobject after it has been deleted on a client no longer results in client attaching a new dynamic subobject.
- Fixed a bug that caused entity pool reservations to cease after a request times out.
- Running `BuildWorker.bat` for `SimulatedPlayer` no longer fails if the project path has a space in it.

> - The editor no longer crashes if you start PIE with out-of-date schema.

Reviewer: ```suggestion
- Fixed a crash when starting PIE with out-of-date schema.
```
